src/tests/ftest/object/object_integrity.py

```diff
@@ -28,6 +28,7 @@
 import ctypes
 import time
 import avocado
+import random
 
 sys.path.append('./util')
```

Comment:
Lines 33-36 (originally 32-25) should be removed.  We should be using PYTHONPATH to define the library locations - which launch.py currently does for you.

Reply:
Have updated script.